dapps/evancore.vue.libs/src/components/dapp-wrapper/mnemonic-export/mnemonic-export.ts

```diff
@@ -45,16 +45,19 @@ export default class MnemonicExport extends mixins(EvanComponent) {
     this.mnemonic = await this.getMnemonic();
     this.now = new Date();
     this.alias = window.localStorage.getItem('evan-alias');
+
+    // Show directly the mnemonic export and do not allow closing.
+    this.mnemonic && this.$nextTick(() => this.showModal());
```

Comment:
Can't this be triggered in some `created()` lifecycle hook instead? Avoid ticks in general whenever possible :)

Reply:
The `created` function is asynchronous and the content is not rendered before mnemonic is loaded. As a result of this, the modal reference is not available at that point or within any other lifecycle hook. Within the next ticket function, the content is rendered and the modal can be opened.